contributr/contriblog/templates/contriblog/detail.html

```diff
@@ -1,14 +1,14 @@
 {% load markdown_deux_tags %}
 <html>
-<head><title>Contriblog</title></head>
+<head><title>ContriBlog</title></head>
 <body>
```

Comment:
Yes, but we need to merge the branches before the blog templates can inherit from the base template. I'm thinking that all the blog templates inherits from a contriblog/base.html, which inherits from contributr/templates/base.html. This will give us more control over the how the blog is displayed on the site.